platform/node/scripts/Dockerfile_Build

```diff
@@ -1,32 +0,0 @@
-FROM ubuntu:focal
```

Comment:
I think we want to keep this dockerfile

Reply:
Absolutely. My plan was to revert it not to delete it 

I will be traveling next 24 hours or so. Could you please unde oth delete?